ui/src/testing/stubs/workspace-service-stub.ts

```diff
@@ -134,13 +134,12 @@ export class WorkspacesServiceStub {
         observer.complete();
       }, 0);
     });
-    return observable;
   }
 
-  public updateWorkspace(workspaceNamespace: string,
-      workspaceId: string,
-      newWorkspace: Workspace): Observable<Workspace> {
-    const observable = new Observable(observer => {
+  updateWorkspace(workspaceNamespace: string,
```

Comment:
Philosophy point (no need to fix): I generally prefer an indent over lining up arguments since I judge that the former is more widely supported by any arbitrary text editor and I want to give developers as much leeway as possible in terms of their choice of text editor.